webpack/assets/javascripts/react_app/routes/Statistics/StatisticsPage/StatisticsLoadingPage/statisticsloadingpage.scss

```diff
@@ -0,0 +1,16 @@
+@import "~patternfly/dist/sass/patternfly/_color-variables.scss";
+
+.statistics-container {
+  display: flex;
+  flex-wrap: wrap;
+
+  #empty-statistics {
```

Comment:
maybe we should call it loader or something that its clear its a temporary loader class? eg empty has another meaning?